rpc/common/moc_common_common.proto

```diff
@@ -76,6 +84,7 @@ message Status {
 	ProvisionStatus provisioningStatus = 2;
 	Error lastError = 3;
 	Version version = 4;
+	DownloadStatus downloadStatus = 5;
```

Comment:
Try to include DownloadStatus within Status.

Reply:
Do you mean to deconstruct the subproperties of downloadStatus into status?

Reply:
You can pack DownloadStatus into Status as an element. Try to pack it in a way so you can unpack it.
There may be other elements in status, so add one for DownloadStatus. This wouldnt give it a struct, but more like arbitrary data

Reply:
Isn't this already the case? The DownloadStatus is part of Status right now